new-player/src/main/java/net/newpipe/newplayer/NewPlayerImpl.kt

```diff
@@ -298,10 +298,6 @@ class NewPlayerImpl(
                         mutableErrorFlow.emit(e)
                     }
                 }
-
```

Comment:
Useful cleanup, but if I'm nitpicky this code change should not be part of this PR.
I don't care too much about this though :P. However, I don't know how other devs feel about this. I'd say do it as you feel best with.